front/app/components/UI/CloseIconButton/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { MouseEvent, KeyboardEvent } from 'react';
+import { IconButton } from '@citizenlab/cl2-component-library';
+
+// i18n
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+
+interface Props {
+  onClick: (event?: MouseEvent | KeyboardEvent) => void;
+  a11y_buttonActionMessage: ReactIntl.FormattedMessage.MessageDescriptor;
+  iconColor: string;
+  iconColorOnHover: string;
+  className?: string;
+  widthInPx?: number;
+  heightInPx?: number;
+}
+
+const CloseIconButton = ({
+  onClick,
+  a11y_buttonActionMessage,
+  intl: { formatMessage },
+  iconColor,
+  iconColorOnHover,
+  className,
+  widthInPx,
+  heightInPx,
+}: Props & InjectedIntlProps) => {
+  return (
+    <IconButton
+      className={className ?? ''}
+      iconName="close"
+      onClick={onClick}
+      a11y_buttonActionMessage={formatMessage(a11y_buttonActionMessage)}
+      iconColor={iconColor}
+      iconColorOnHover={iconColorOnHover}
+      widthInPx={widthInPx}
+      heightInPx={heightInPx}
+    />
+  );
+};
+
+export default injectIntl(CloseIconButton);
```

Comment:
Turns out there's not a much benefit to have this as a separate component, except that we don't need to have formatMessage in every component that imports CloseIconButton (which is 3 out of 5 atm). And the iconName which is the same.

Reply:
Still good to have it, I think, will be useful in the future as well, hopefully..